src/assets/models/input-stat-claim-accredited.model.ts

```diff
@@ -0,0 +1,23 @@
+/* istanbul ignore file */
+
+import { ApiProperty } from '@nestjs/swagger';
+import { ClaimType } from '@polymeshassociation/polymesh-sdk/types';
+
+import { InputStatClaimModel } from '~/assets/models/input-stat-claim.model';
+
+export class InputStatAccreditedClaimModel extends InputStatClaimModel {
```

Comment:
```suggestion
export class StatAccreditedClaimModel extends StatClaimModel {
```

To me this name is confusing since models aren't supposed to be Input. I think it would be fine without the qualifier